py_trees/common.py

```diff
@@ -67,6 +67,15 @@ def __init__(self, synchronise=True):
             """
             super().__init__(synchronise=synchronise)
 
+        def __str__(self) -> str:
+            """
+            Human readable description.
+            """
+            description = "--" + self.__class__.__name__ + "("
+            description += u'\u26A1' if self.synchronise else "-"  # lightning bolt
```

Comment:
BTW although I like lightning bolts, I was wondering if you would like to consider this instead 'repeat': https://en.wiktionary.org/wiki/%F0%9F%94%81

Reply:
Might be misleading? i.e. looks like it has them on repeat...which is the opposite of pausing some behaviours. I wouldn't mind a better idea than a 'cool' bolt though...

Reply:
Noted. I got the inspiration from Dropbox's synchronize status.

Reply:
Ah, that makes sense for that, since it synchronises to and from the cloud.

https://www.fileformat.info/info/unicode/char/21c8/index.htm?

Reply:
Not a great choice, that would suit better the opposite, everything SUCCESS together.

Reply:
hmm.. It could also mean parallel. I think the lightning is fine because it's just a symbol you don't expect, and when you look at it you're like - oh, this is what this means, okay.